packages/amplify_storage_s3/ios/Classes/FlutterDownloadFileRequest.swift

```diff
@@ -26,15 +26,14 @@ struct FlutterDownloadFileRequest {
         self.options = setOptions(request: request)
     }
     
-    static func isValid(request: Dictionary<String, AnyObject>) -> Bool {
-        var valid: Bool = true;
+    static func isValid(request: Dictionary<String, AnyObject>) throws {
+        let validationErrorMessage = "Download request malformed."
         if !(request["key"] is String) {
-            valid = false
+            throw AmplifyFlutterValidationException(errorDescription: validationErrorMessage, recoverySuggestion: "key is missing.")
         }
```

Comment:
Same as android around renaming the validation methods.

Reply:
Done.